graphql/schemabuilder/pagination.go

```diff
@@ -42,6 +43,21 @@ type ConnectionArgs struct {
 	Args   interface{}
 }
 
+// PaginationArgs are embedded in a struct
+type PaginationArgs struct {
+	First  *int64
+	Last   *int64
+	After  *string
+	Before *string
+}
+
+//PaginationInfo can be returned in a PaginateFieldFunc
```

Comment:
nit: space at beginning, period at the end

Reply:
consider adding a comment similar to https://github.com/graphql/graphql-relay-js/issues/94#issuecomment-232410564 about how to populate this struct.